routes/users.js

```diff
@@ -29,6 +28,12 @@ userRouters
     SignupValidation.validatePassword,
     UserController.updatePassword
   )
+  .put(
```

Comment:
The URL should be shortened to `/users/:username/password`

Reply:
It is conflicting with another endpoint for resetting password called `/users/: token/password`
@oesukam